File: Bridge/Resources/.generated/system/runtime/compilerServices/formattableStringFactory.js
    Bridge.define("System.Runtime.CompilerServices.FormattableStringFactory", {
        statics: {
            methods: {
                create: function (format, args) {
                    if (args === void 0) { args = []; }
                    return new System.FormattableStringImpl(format, args);
                }
            }
        }
    });

File: Bridge/System/CompilerServices/FormattableStringFactory.cs
namespace System.Runtime.CompilerServices
{
    [Bridge.Convention(Member = Bridge.ConventionMember.Field | Bridge.ConventionMember.Method, Notation = Bridge.Notation.CamelCase)]
    public static class FormattableStringFactory
    {
        public static FormattableString Create(string format, params object[] args)
        {
            return new FormattableStringImpl(format, args);
        }
    }
}
